webapp/src/components/SignInPage/SignInPage.js

```diff
@@ -14,32 +13,26 @@ export default class SignInPage extends React.PureComponent {
   static propTypes = {
     isLoading: PropTypes.bool.isRequired,
     isConnected: PropTypes.bool.isRequired,
-    onConnect: PropTypes.func.isRequired,
     onNavigate: PropTypes.func.isRequired
   }
 
   componentWillReceiveProps(nextProps) {
     if (nextProps.isConnected) {
-      this.props.onNavigate(locations.parcelMapDetail(0, 0))
+      // this.props.onNavigate(locations.parcelMapDetail(0, 0))
+      console.log(locations)
```

Comment:
Woops

Reply:
goddamn! I was sure I had reverted this, super thanks